app/controllers/registered_applications_controller.rb

```diff
@@ -4,6 +4,11 @@ def index
   end
 
   def show
+    begin
+      @application = current_user.applications.find(params[:id])
```

Comment:
@jeffminnear good protection of user-owned data with the `current_user`-scoped query :+1: